client/src/app/site/base/base-repository.ts

```diff
@@ -134,7 +138,7 @@ export abstract class BaseRepository<V extends BaseViewModel, M extends BaseMode
      * return the Observable of the whole store
      */
     public getViewModelListObservable(): Observable<V[]> {
-        return this.viewModelListSubject.asObservable();
+        return this.viewModelListSubject.asObservable().pipe(auditTime(1));
```

Comment:
requires documentation (esp. the audit time reasoning)

Reply:
Done above.